pkg/test/kube/accessor.go

```diff
@@ -49,7 +49,7 @@ const (
 
 var (
 	defaultRetryTimeout = retry.Timeout(time.Minute * 10)
-	defaultRetryDelay   = retry.Delay(time.Second * 10)
+	defaultRetryDelay   = retry.Delay(time.Second * 1)
```

Comment:
Right now we only retry things like `getPod` and `chckReady` every 10s.  This adds up quickly. A really bad example is: Test starts.. 1s goes by and pod gets ready ... 9s go by and we check isReady.. 1s goes by and the readiness probe marks it ready... 9s more and we call isReady again. In reality we should have been ready after 1s, but due to a bad combination of probing internal and retry interval, we actually take 19s.

Retrying every 1s seems pretty reasonable and unlikely to put too much load on api server